external/src/main/java/com/redhat/cloud/policies/engine/AlertStarter.java

```diff
@@ -64,56 +58,21 @@ void startApp(@Observes StartupEvent startup) {
         initialize();
     }
 
-    private void printEnvVars() {
-        LOGGER.info("=== ENVIRONMENT VARIABLES - START ===");
-        for (Map.Entry<String, String> var : System.getenv().entrySet()) {
-            String value = isWhitelisted(var.getKey()) ? var.getValue() : "NOT_WHITELISTED";
-            LOGGER.info(var.getKey() + "=" + value);
-        }
-        LOGGER.info("=== ENVIRONMENT VARIABLES - END ===");
-        LOGGER.info("=== QUARKUS CONFIG - START ===");
-        for (String key : ConfigProvider.getConfig().getPropertyNames()) {
-            if (key.startsWith("engine")) {
-                ConfigProvider.getConfig().getOptionalValue(key, String.class).ifPresent(value -> {
-                    LOGGER.info(key + "=" + value);
-                });
-            }
-        }
-        LOGGER.info("=== QUARKUS CONFIG - END ===");
-    }
-
-    private static final List<String> PREFIX_WHITELIST = List.of(
-            "QUARKUS_LOG_CLOUDWATCH_LOG_STREAM_NAME",
-            "ENGINE",
-            "HOSTNAME"
-    );
-
-    private boolean isWhitelisted(String key) {
-        for (String prefix : PREFIX_WHITELIST) {
-            if (key.startsWith(prefix)) {
-                return true;
-            }
-        }
-        return false;
-    }
-
     void stopApp(@Observes ShutdownEvent shutdown) {
         LOGGER.info("Shutting down Policies Engine.");
         alerts.stop();
     }
 
     void initialize() {
         // PluginRegister is already using CDI, but we want it to initialize after Alerts has started
-        alerts.init()
-                .whenComplete((empty, error) -> {
-                    if(error != null) {
-                        LOGGER.error("Engine start failed", error);
-                        alerts.stop();
-                    } else {
-                        pluginRegister.init();
-                        ((StatusServiceImpl) statusService).setStarted(true);
-                        LOGGER.info("Started Policies Engine");
-                    }
-        }).join();
```

Comment:
This was nothing more than a sequential execution using a thread from `ForkJoinPool`.